app/views/users/_tabs.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row">
```

Comment:
You only needed to add this code to the users/show.html.erb, I believe the tabs, feed, followers, following, and liked html pages are for the next PR for tabbed interface. Good job implementing it but make sure you aren't adding in anything in this PR thats for the next one

Reply:
I think it's showing up here because Anna accidentally merged my pull request for the tabbed interface before this one was merged, but I will keep that in mind.